protocol/common/message_linkpreview.go

```diff
@@ -468,3 +470,26 @@ func (m *Message) ConvertFromProtoToStatusLinkPreviews(makeMediaServerURL func(m
 
 	return previews
 }
+
+func decompressPublicKey(input string) string {
```

Comment:
@igor-sirotin, should we use something like this instead? Do we need the new custom implementation?

https://github.com/status-im/status-go/blob/85f8c92cde612820f9ee960b246ab3de285bc644/protocol/common/message_sender.go#L428

Reply:
We have so many implementations of these, `DecompressPubkey` is not the only one 🙁 
I just went with the simplest solution for now, because it needs clean up everywhere anyway.

Also, `DecompressPubkey` returns `ecdsa.PublicKey`, which I will have to convert back to String after operations. I don't know if that's a difficult operation, but this code looked simpler to me.

Reply:
Yeah, up to you really @igor-sirotin. It might be worth trying, because calling a simple function that's already implemented and converting to a string might be easier than having a custom implementation and adding the unit tests.